src/transferlistwidget.cpp

```diff
@@ -424,6 +424,19 @@ void TransferListWidget::copySelectedMagnetURIs() const
     qApp->clipboard()->setText(magnet_uris.join("\n"));
 }
 
+// create list of captions from tasklist.
+void TransferListWidget::copySelectedCaptions() const
```

Comment:
I kept as much of the original submission from chrishirst with the least change possible. You're right, it's not important. The function still works the same :smile: